### PR TITLE
[common] Comments refinement

### PR DESCRIPTION
**Pull request by yskuridi**
Issue: MDP-67484
Test: Not needed

### CODE REVIEW

**Review by dmitryermilov — APPROVED** [review]

**Review by dgurulev — DISMISSED** [review]

File: _studio/shared/src/libmfx_allocator_vaapi.cpp [hunk 1]
```diff
@@ -947,7 +947,7 @@ std::pair<mfxHDL, mfxResourceType> mfxFrameSurface1_hw_vaapi::GetNativeHandle()
 {
     std::shared_lock<std::shared_timed_mutex> guard(m_hdl_mutex);
 
-    // TODO: temporary changed returned signature, return back (uncomment subsequent line) since current version violates the spec
+    // temporary changed returned signature, return back (uncomment subsequent line) since current version violates the spec
```
Comment (mgonchar) [inline _studio/shared/src/libmfx_allocator_vaapi.cpp:950]: please, delete these two lines  (950 and 951) completely

File: _studio/shared/include/libmfx_core.h [hunk 2]
```diff
@@ -417,7 +417,7 @@ class CommonCORE20 : public deprecate_from_base<CommonCORE>
 
     virtual void* QueryCoreInterface(const MFX_GUID &guid)                    override;
 
-    // TODO: check if we allowed to break vtable for MSDK2.0
+    // check if we allowed to break vtable for MSDK2.0
```
Comment (mgonchar) [inline _studio/shared/include/libmfx_core.h:420]: remove this ine completely

**Review by mgonchar — CHANGES_REQUESTED** [review]

**Comment by mgonchar** [comment]
@yskuridi please do not forget to align CS with this version as well

File: _studio/shared/src/libmfx_allocator_vaapi.cpp (hunk 1 above)
Comment (yskuridi) [reply _studio/shared/src/libmfx_allocator_vaapi.cpp]: Fixed

File: _studio/shared/include/libmfx_core.h (hunk 2 above)
Comment (yskuridi) [reply _studio/shared/include/libmfx_core.h]: Fixed

**Review by dmitryermilov — APPROVED** [review]

**Review by mgonchar — APPROVED** [review]

**Pull request merged**